src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -184,6 +185,25 @@ private static List<DetailNode> getAllNewlineNodes(DetailNode descriptionNode) {
         return textNodes;
     }
 
+    /**
+     * Checks if the given description node is part of a block Javadoc tag.
+     *
+     * @param description the node to check
+     * @return {@code true} if the node is inside a block tag, {@code false} otherwise
+     */
+    private static boolean isBlockDescription(DetailNode description) {
+        boolean isBlock = false;
+        DetailNode currentNode = description;
+        while (currentNode != null) {
+            if (currentNode.getType() == JavadocTokenTypes.JAVADOC_TAG) {
```

Comment:
There are two types of tags, block and inline,.
I think they both use this token. Block tags are tags with special names as far I remember